src/test/java/com/birdie/kafka/connect/smt/DebeziumJsonDeserializerTest.java

```diff
@@ -229,6 +229,19 @@ public void transformsEmptyObject() {
         assertEquals(Schema.Type.STRUCT, jsonSchema.type());
     }
 
+    @Test
+    public void transformsEmptyString() {
+        Struct value = new Struct(simpleSchema);
+        value.put("id", "1234-5678");
+        value.put("json", "");
```

Comment:
If this was the value, I'd agree :)

```suggestion
        value.put("json", "\"\"");
```